XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -57,44 +57,69 @@
         </DataTemplate>
 
-        <DataTemplate x:Key="ContactListViewTemplate2" x:DataType="local1:Contact">
-            <Grid Margin="0,12,0,12">
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" MinWidth="550"/>
-                </Grid.ColumnDefinitions>
-                <TextBlock Grid.Column="1"
-                           Text="{x:Bind Name}"
-                           x:Phase="1"
-                           Margin="0,12,0,0"/>
+        <DataTemplate x:Key="MessageViewTemplate" x:DataType="local1:Message">
+            <Grid Height="Auto" Margin="4" HorizontalAlignment="{x:Bind MsgAlignment}">
+                <StackPanel MinHeight="75" Width="350" Padding="10, 0, 0, 10" Background="{x:Bind BgColor}" CornerRadius="4">
+                    <TextBlock Text="{x:Bind MsgText}" Padding="0, 10, 0, 0" FontSize="20" Foreground="{ThemeResource SystemControlForegroundAltHighBrush}"/>
+                    <TextBlock Text="{x:Bind MsgDateTime}" Padding="0, 0, 0, 10" FontSize="15" Foreground="{ThemeResource SystemControlForegroundAltMediumHighBrush}"/>
+                </StackPanel>
             </Grid>
         </DataTemplate>
+
+        <DataTemplate x:Key="BasicListViewTemplate" x:DataType="local1:Contact">
+            <TextBlock Text="{x:Bind Name}" x:Phase="1" Margin="0,5,0,5"/>
+        </DataTemplate>
+        
     </Page.Resources>
     <StackPanel>
-        <local:ControlExample HeaderText="A Simple ListView with Selection support">
-            <ListView
-                x:Name="Control2"
-                CanDragItems="True"
-                CanReorderItems="True"
-                AllowDrop="True"
-                BorderThickness="1"
-                BorderBrush="DimGray"
-                MinWidth="550" 
-                Height="400"/>
+        
+        <local:ControlExample HeaderText="Basic ListView with Simple DataTemplate"
+                              XamlSource="ListView\ListViewSample1_xaml.txt"
+                              CSharpSource="ListView\ListViewSample1_cs.txt">
+            <StackPanel>
+                <TextBlock Margin="0,0,0,15">
+                    This is a basic ListView that has the full source code below. <LineBreak/>Other samples on this page display only the additional markup needed to customize a simple ListView like this one.
+                </TextBlock>
+                <ListView x:Name="BaseExample"
+                    ItemTemplate="{StaticResource BasicListViewTemplate}"
+                    BorderThickness="1"
+                    BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}"
+                    Width="350" 
+                    Height="400"
+                    HorizontalAlignment="Left"/>
+            </StackPanel>
+        </local:ControlExample>
+
+        <local:ControlExample HeaderText="ListView with Selection Support">
+            <StackPanel>
+                <TextBlock TextWrapping="Wrap" Margin="0,0,0,15">
```

Comment:
Consider breaking up this TextBlock into either multiple TextBlocks, or into a RichTextBlock with Runs, or just into new lines. As it is, this chunk of code is difficult to read with inline line breaks and formatting.